site/docs/guidelines/inclusive_design.mdx

```diff
@@ -0,0 +1,70 @@
+
+## Best practises
+While designing inclusive and accessible services may seem a daunting task, it gets easier when you take it into account from the beginning of the project. Here is a list of best practises from the city of Helsinki projects.
+
+### Imagery
+Most services in Helsinki use some kind of imagery either for conveying information or adding visual interest. Using imagery always requires some extra thought to ensure all users can feel welcomed and included in your service.
+
+If you are using photographs, make sure you intentionally diversify your images. This means ensuring that a wide array of different users are represented in your imagery.
+
+<p>Using abstract images is another way of improving the inclusiveness of your images. Instead of using realistic and detailed photographs, your project can use more conceptual illustrations. <Link href="https://brand.hel.fi/kuvitusohjeisto/" external>The Helsinki Visual identity guideline</Link> has a wide array of abstract image materials to be used in your project.</p>
+
+### Writing copy
+Using inclusive language in your service is as important as using inclusive imagery. Text often contains the most important information of your service and it is accessible to most user groups in some way or another. 
+
+Prefer using simple words and short sentences to keep your copy as easy to understand as possible. Using infrequently used words and long sentences make the copy more difficult to understand for many people - especially if they are not fluent in the main language of your interface.
+
+Provide your user interface in multiple languages. In general, at least Finnish, Swedish and English should be available. Depending on the users of your service you may need to consider providing other languages too.
+
+### Making forms
+Forms in their different variations are very common in the city of Helsinki services. They are often used to collect sensitive information such as user's name, gender, age, etc. When designing forms, it is important to consider:
+
+- Why are we asking a specific question? Is this data needed?
+- How the question is asked? What options can the user choose from?
+
+Generally, if the data is not needed, the question should not be included in the form. Data should not be asked "just in case" either. If you do not know whether if need the data or not, it should not be asked. It is a good practise providing a short explanation to the user why the data is being asked and how it will be used.
+
+If the data is needed, next you need to decide how the question is asked and what options are offered to the user. Often giving a set of mutually exclusive options to choose from can make the user feel excluded. In these cases, it may be better to allow the user to freely type the answer if an inclusive set of options cannot be determined.
```

Comment:
what means mutually exclusive?

Reply:
Only one option is possible - or two or more options cannot be set at the same time. 

But, maybe this is not very accessible language and it can be expressed in much more understandable way. I will fix this.